R/backend-common.R

```diff
@@ -245,95 +161,37 @@ set_audio_backend <- function(backend) {
 #' Loads an audio file from disk using the default loader (getOption("torchaudio.loader")).
 #'
 #' @param filepath (str): Path to audio file
-#' @param offset (int): Number of frames (or seconds) from the start of the file to begin data loading. (Default: ``0``)
-#' @param duration (int): Number of frames (or seconds) to load.  0 to load everything after the offset. (Default: ``0``)
+#' @param offset (int): Number of frames (or seconds) from the start of the file to begin data loading. (Default: `0`)
+#' @param duration (int): Number of frames (or seconds) to load.  `-1` to load everything after the offset. (Default: `-1`)
 #' @param unit (str): "sample" or "time". If "sample" duration and offset will be interpreted as frames, and as seconds otherwise.
 #'
 #'
 #' @export
-torchaudio_loader <- function(
-  filepath,
-  offset = 0L,
-  duration = 0L,
-  unit = c("samples", "time")
-) {
+torchaudio_load <- function(
+    filepath,
+    offset = 0L,
+    duration = -1L,
+    unit = c("samples", "time")) {
   loader <- getOption("torchaudio.loader", default = tuneR_loader)
```

Comment:
Isn't the default supposed to be `av`? I think it's probably better to have a wrapper function `get_torchaudio_loader()` and use it whenever you need to query that option. This way you only need to set the default once.

Reply:
(((addressing this together with the other loading-related comment, so closing)))